src/MutableMap/MutableMapTrait.dfy

```diff
@@ -0,0 +1,66 @@
+/*******************************************************************************
+*  Copyright by the contributors to the Dafny Project
+*  SPDX-License-Identifier: MIT
+*******************************************************************************/
+
+// RUN: %dafny /compile:0 "%s"
+
+include "../Wrappers.dfy"
+
+module {:options "-functionSyntax:4"} MutableMapTrait {
+  import opened Wrappers
+
+  trait {:termination false} MutableMapTrait<K(==),V(==)> {
```

Comment:
Add a comment that this is duplicated here for the benefit of the feasibility implementation.